### PR TITLE
Added option to make payload relative to a path

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

File: tasks/sri.js [hunk 1]
```diff
@@ -136,6 +136,11 @@ task = function () {
         // Build manifest using the "reduce to object" pattern
         function (manifest, file, callback) {
             generate(file.src, options, function (err, sriData) {
+                //Make relative if a cwd is specified
```
Comment (XhmikosR) [inline tasks/sri.js]: Please add a space after `//`.

File: Gruntfile.js [hunk 2]
```diff
@@ -46,6 +46,19 @@ module.exports = function (grunt) {
             ]
         },
 
+        // Create a second manifest with relative paths
+        "cwd": {
+            "options": {
+                "dest": "./tmp/cwd.json",
```
Comment (XhmikosR) [inline Gruntfile.js]: Please remove this unneeded trailing space.

**Review by XhmikosR — CHANGES_REQUESTED** [review]
After those 2 minor changes we can merge this, thanks!

File: Gruntfile.js (hunk 2 above)
Comment (LaurentGoderre) [reply Gruntfile.js]: Where is the trailing whitespace? I tried to find it but I can't. I used the trim function in Atom as well and no change

Comment (XhmikosR) [reply Gruntfile.js]: Sorry, I wanted to say **comma**.

**Comment by LaurentGoderre** [comment]
Done!

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
Thanks! I'll wait a few days for @neftaly and merge this.

If I forget about it please ping me.

**Comment by LaurentGoderre** [comment]
Thanks!

**Pull request merged**